client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -1,37 +1,42 @@
 import { Button, Gridicon } from '@automattic/components';
+import classnames from 'classnames';
 import { translate, useRtl } from 'i18n-calypso';
 import Badge from 'calypso/components/badge';
 import { isTaskDisabled } from './task-helper';
 import { Task } from './types';
 
 const ChecklistItem = ( { task }: { task: Task } ) => {
 	const isRtl = useRtl();
-	const { id, isCompleted, actionUrl, title, actionDispatch } = task;
+	const { id, isCompleted, keepActive, actionUrl, title, actionDispatch } = task;
 	const action = actionDispatch ? { onClick: actionDispatch } : { href: actionUrl };
 	const taskDisabled = isTaskDisabled( task );
 
 	return (
-		<li className={ `launchpad__task-${ id }` }>
+		<li
+			className={ classnames( 'launchpad__task', {
+				'keep-active': keepActive,
+				'is-completed': isCompleted,
+			} ) }
+		>
 			<Button
 				className="launchpad__checklist-item"
 				disabled={ taskDisabled }
 				href={ actionUrl }
 				data-task={ id }
 				{ ...action }
 			>
-				{ isCompleted && taskDisabled && (
-					<div className="launchpad__checklist-item-status">
+				{ isCompleted && (
```

Comment:
I know we removed `&& taskDisabled &&` in response to @jeyip's comment above. But @agrullon95 I'd asked about this last line week and it sounded like that extra check was needed. Can you confirm it's ok to remove now?

Reply:
We may not need taskDisable anymore. I'm going to make some changes locally to test it out. We can create a follow PR for these changes.